intervaltree/node.py

```diff
@@ -375,6 +375,13 @@ def get_new_s_center():
             child = Node(new_x_center, get_new_s_center())
             self.s_center -= child.s_center
 
+            # Move any overlaps into child
+            for iv in set(self.s_center
+                    | (self[0].s_center if self[0] else set())):
+                if iv.contains_point(child.x_center):
+                    self.s_center.discard(iv)
```

Comment:
This will end up not removing `iv` from `self[0]`, and will break our invariants.